javascripts/discourse/components/user-card-meta-data.gjs

```diff
@@ -6,8 +6,34 @@ import i18n from "discourse-common/helpers/i18n";
 export default class UserCardMetaData extends Component {
   @service currentUser;
 
+  get uniqueId() {
+    return this.args.outletArgs.user["ai_stream_conversation_unique_id"];
+  }
+
+  get username() {
+    if (this.uniqueId) {
+      return this.uniqueId.split("/u/")[1];
+    }
+  }
+
+  get siteName() {
+    if (this.uniqueId) {
+      return this.uniqueId.split("/u/")[0].split("https://")[1];
+    }
+  }
+
   <template>
     {{#if this.currentUser.admin}}
+      {{#if this.uniqueId}}
+        <a href={{this.uniqueId}}
+          target="_blank"
+          rel="noopener noreferrer"
+          class="user-card-meta__profile-link"
+        >
+          {{icon "up-right-from-square"}}
+          {{this.username}} on {{this.siteName}}
```

Comment:
We might want to translate this

Reply:
on ask? this is only used internally at the moment ... but yeah eventually